README.md

```diff
@@ -170,5 +212,7 @@ Write what an example ERB file (aka view) might look like to list all the warrio
 
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
-<!-- code here -->
+<% @warriors.each do |warrior| %>
+<%= warrior.name %>
+<% end %>
```

Comment:
in a real application we would want to contain the warrior names in some kind of `html` tag.